spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/cql/Schema.scala

```diff
@@ -138,7 +149,9 @@ case class TableDef(
     clusteringColumns: Seq[ColumnDef],
     regularColumns: Seq[ColumnDef],
     indexes: Seq[IndexDef] = Seq.empty,
-    isView: Boolean = false) extends StructDef {
+    isView: Boolean = false,
+    clusteringOrder: Option[Seq[ClusteringOrder]] = None,
```

Comment:
Shouldn't this be a property of the ColunDef now? Ie do you really need to specify it separately?